src/ray/protobuf/pubsub.proto

```diff
@@ -47,8 +47,6 @@ enum ChannelType {
   RAY_ERROR_INFO_CHANNEL = 8;
   /// A channel for logs from various Ray components.
   RAY_LOG_CHANNEL = 9;
-  /// A channel for keys to pickled python functions and actor classes.
```

Comment:
`reserve` the tag number for now, so it is easy for others to see the tag number has been deleted, and they can choose to re-use / not-use the value?

Reply:
Given that backward compatibility is not a critical issue right now, are you ok with just adjusting the labels?

Reply:
Yeah this is minor. Reducing tag numbers of other fields by 1 is fine too.